modules/mauth-authenticator/src/main/java/com/mdsol/mauth/RequestAuthenticator.java

```diff
@@ -191,6 +180,7 @@ private boolean fallbackValidateSignatureV1(MAuthRequest mAuthRequest, PublicKey
           mAuthRequest.getResourcePath(),
           mAuthRequest.getQueryParameters()
       );
+      mAuthRequestV1.setInputStream(mAuthRequest.getInputStream());
```

Comment:
if `fallbackValidateSignatureV1` isn't supposed to be called with an InputStream body, we should thrown an exception because it indicates a bug in MAuth.

Reply:
updated in https://github.com/mdsol/mauth-jvm-clients/pull/132/commits/3c0b8578c2ea43e3c379b8d895c7d551edda12a8